src/DocRaptor.php

```diff
@@ -155,11 +160,12 @@ public function setBaseUrl($base_url)
      *
      * @param bool|string $filename
      * @return bool|mixed
+     * @throws Exception
      */
     public function fetchDocument($filename = false)
     {
         if ($this->api_key != '') {
-            $url = $this->url_protocol . '://docraptor.com/docs?user_credentials=' . $this->api_key;
+            $url = $this->url_protocol . '://'. $this->api_url . '?user_credentials=' . $this->api_key;
```

Comment:
Another possibility: api key is in the `$fields` array below and gets handled by `http_build_query`